src/components/CreateResetPassword/components/ChangePasswordConfirmation/ChangePasswordConfirmation.css

```diff
@@ -0,0 +1,23 @@
+@import "@folio/stripes-components/lib/variables.css";
+
+.wrap {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+.centered {
+  position: relative;
+  max-width: 500px;
+  width: 100%;
```

Comment:
div has 100% width by default. Can we remove this line?

Reply:
Sure, missed that